app/api/tax.py

```diff
@@ -0,0 +1,75 @@
+from flask_rest_jsonapi import ResourceDetail, ResourceList, ResourceRelationship
+from marshmallow_jsonapi.flask import Schema, Relationship
+from marshmallow_jsonapi import fields
+from app.models import db
+from app.api.helpers.permissions import jwt_required, is_admin, is_user_itself
+from sqlalchemy.orm.exc import NoResultFound
+from flask_rest_jsonapi.exceptions import ObjectNotFound
+from app.models.event import Event
+from app.models.tax import Tax
+
+class TaxSchema(Schema):
+
+    class Meta:
+        type_ = 'tax'
+        self_view = 'v1.tax_detail'
+        self_view_kwargs = {'id': '<id>'}
+
+    id = fields.Str(dump_only=True)
+    country = fields.Str()
+    tax_name = fields.Str(required=True)
+    tax_rate = fields.Float(required=True)
+    tax_id = fields.Str(required=True)
+    is_invoice_sent = fields.Boolean(default=False)
```

Comment:
to be called `should_send_invoice`. `is_invoice_sent` is mis-leading. Please refactor db table name too.